cmd/consume.go

```diff
@@ -54,8 +57,48 @@ func consume(messageType string, uri string, tenant string) error {
 	fmt.Println()
 
 	opts := make([]amqp.ConnOption, 0)
+
+	//Enable TLS
+	if insecure {
+		opts = append(opts, amqp.ConnTLSConfig(createTlsConfig()))
+	}else{
+		//Read cert from local TLS file 
+    	    	caCert, err := ioutil.ReadFile("tls.crt")       
```

Comment:
To my understanding this would mean that there **must** be a `tls.crt` file for this to work. Shouldn't it use the `tlsRoute` or better `tlsPath` here. And only if the argument was set. Because IIRC, otherwise it would use the system-wide CA certs, which might be fine too.

Reply:
Yep, this was a typo I had missed :)